tests/flow/test_concurrent_query.py

```diff
@@ -49,6 +50,19 @@ def thread_run_query(graph, query, threadID):
     global assertions
     assertions[threadID] = graph.query(query)
 
+def thread_try_run_query(graph, query, threadID):
+    global assertions
+    try:
+        assertions[threadID] = graph.query(query)
+    except ResponseError as e:
+        assertions[threadID] = str(e)
```

Comment:
combine thread_run_query and thread_try_run_query into a single function, which runs the query and updates a global exception array in case of an exception, such that we would have 2 global arrays (assertions and exceptions)

Reply:
done